views/default3.handlebars

```diff
@@ -2112,120 +2206,13 @@
             }
             args.hide = hide;
             QV('uiViewButton5', !(args.hide & 4)); // Hide the footer toggle button if footer is hidden anyway.
+            QV('toggleFooterMenuItem', !(args.hide & 4)); // Hide the footer toggle menu item if footer is hidden anyway.
+            QV('toggleNightMenuItem', ((features2 & 0x00100000) == 0) && ((features2 & 0x00200000) == 0)); // Hide night mode toggle if permanent night (1) or day (2) mode is set
+            QV('toggleModernUIMenuItem', ((features2 & 0x80000000) == 0)); // Show only if allowed and not forced siteStyle=3
```

Comment:
i think this shouldnt be forced.
i spoke to a gent today who asked if it was possible to set the web ui to moden ui by default
but then let the staff return to the classic ui if they wanted
with this your are FORCING the style regardless and the staff have no way to return to the classic ui if they choice

Reply:
A theme style should be a default you can set in the server config (and at some point we'll change the schema/default from classic to modern unless already defined in config). 

Then each use should be able to select in the menu their desired theme and it be remembered for their login.